recipes/cyvlfeat/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "cyvlfeat" %}
+{% set version = "0.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 95e63e4451a0368aa26fcc45175b36d3990c271329855ea155b687dc6ca26f57
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [win]
+
+requirements:
+  build:
+     - {{ compiler('c') }}
+
+  host:
+    - python
+    - pip
+    - numpy
+    - cython
+    - vlfeat
```

Comment:
Does vlfeat have `run_exports` configured?

Reply:
It does not, but the recipe has also not been updated since Sept 2018: https://github.com/conda-forge/vlfeat-feedstock/blob/master/recipe/meta.yaml

Reply:
But vlfeat doesn't adhear to semver so any version change tends to break compatibility